modules/media/ajax/FileUpload.php

```diff
@@ -107,9 +107,14 @@ function uploadFile()
         showError("Please fill in all required fields!");
         return;
     }
+    $fileName  = str_replace(' ', '_', trim($_FILES["file"]["name"]));
```

Comment:
Why is this string replacement required? I can't tell if it needs to match the javascript below or not (if it does, it currently isn't.. the javascript replaces any number of whitespace characters with `_`, this only replaces a single quote character with `_`)